packages/fluent/scss/appbar/_layout.scss

```
@@ -0,0 +1,118 @@
+@use "_variables.scss" as *;
+@use "../core/color-system" as *;
+
+@mixin kendo-appbar--layout() {
+
+    // Appbar
+    .k-appbar {
+        margin-inline: var(--kendo-appbar-margin-x, #{$kendo-appbar-margin-x});
```

Add bracket spacing to all variables:

```suggestion
        margin-inline: var( --kendo-appbar-margin-x, #{$kendo-appbar-margin-x} );
```